src/commands/create_webhook.rs

```diff
@@ -30,6 +32,6 @@ pub async fn main(args: CreateWebhookArgs, config: &Config) -> Result<()> {
     )?;
     let client = Client::new();
     let response = client.post(url).send().await?.text().await?;
-    println!("{}", response);
+    serde_json::to_writer_pretty(io::stdout().lock(), &response)?;
```

Comment:
Does it make sense to do info!() instead of printing directly to stdout?

Reply:
Hmm, this one could indeed get info'd. I'd leave printing in `get-webhooks`/`get-boards` though